### PR TITLE
FOUR-11972  Fixed a bug when reading the value from the timer event

### PR DESCRIPTION
**Pull request by josechirivella**
## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-11972

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
...

### CODE REVIEW

File: src/components/nodes/boundaryTimerEvent/index.js [hunk 1]
```diff
@@ -15,13 +15,9 @@ export const setEventTimerDefinition = (moddle, node, type, body) => {
     [type]: moddle.create('bpmn:Expression', { body }),
   };
 
-  const eventDefinitions = [
+  return [
     moddle.create('bpmn:TimerEventDefinition', eventDefinition),
   ];
-
-  eventDefinitions[0].id = node.definition.get('eventDefinitions')[0].id;
```
Comment (josechirivella) [inline src/components/nodes/boundaryTimerEvent/index.js:22]: this was the main culprit of the bug

**Review by boliviacoca — APPROVED** [review]

**Review by danloa — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-bd32554f53.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-bd32554f53.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-bd32554f53.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-bd32554f53.eng.processmaker.net

**Pull request merged**